fleetspeak/src/common/proto/fleetspeak_monitoring/resource.proto

```diff
@@ -14,6 +14,10 @@ message AggregatedResourceUsage {
   double max_system_cpu_rate = 4;
   double mean_resident_memory = 5;
   int64 max_resident_memory = 6;
+  double mean_io_read = 7;
```

Comment:
Following on the logic of the previous comments, I'd suggest naming it `mean_io_read_bytes`. Same for other added fields.